File: cppfastlogging/examples/console_static.cpp
#include "h/cppfastlogging.hpp"

using namespace logging;

// File: console_static.cpp
//
// Sample library usage.
int main(void)
{
    Logging logging = Init();
    logging.trace("Trace Message");
    logging.debug("Debug Message");
    logging.info("Info Message");
    logging.success("Success Message");
    logging.warn("Warning Message");
    logging.error("Error Message");
    logging.fatal("Fatal Message");
    return 0;
}

File: fastlogging/examples/console_static.rs
use fastlogging::{Logging, LoggingError};

fn main() -> Result<(), LoggingError> {
    let mut logger = Logging::default();
    logger.trace("Trace Message")?;
    logger.debug("Debug Message")?;
    logger.info("Info Message")?;
    logger.success("Success Message")?;
    logger.error("Error Message")?;
    logger.fatal("Fatal Message")?;
    logger.shutdown(false)?;
    Ok(())
}
